src/components/TeamMemberCard/index.tsx

```diff
@@ -1,11 +1,69 @@
 import React from "react";
+import {
+  Card,
+  CardContent,
+  Typography,
+  Tooltip,
+  IconButton,
+} from "@mui/material";
+
+import GitHubIcon from "@mui/icons-material/GitHub";
+import EmailIcon from "@mui/icons-material/Email";
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
 
 interface TeamMemberCardProps {
   memberName: string;
+  github: string;
+  email: string;
+  linkedIn: string;
 }
 // TODO replace with new type
-const TeamMemberCard: React.FC<TeamMemberCardProps> = ({ memberName }) => {
-  return <div className="team-member-card">{memberName}</div>;
+const TeamMemberCard: React.FC<TeamMemberCardProps> = ({
+  memberName,
+  github,
+  email,
+  linkedIn,
+}) => {
+  return (
+    <div className="team-member-card">
+      <Card>
+        <CardContent>
+          <Typography variant="h5">{memberName}</Typography>
+        </CardContent>
+        <Tooltip title="GitHub Profile">
+          <IconButton
+            onClick={() => {
+              window.open(github);
+            }}
+          >
+            <GitHubIcon />
+          </IconButton>
+        </Tooltip>
+        <Tooltip title="Email me. But this doesn't work yet. Sadness.">
+          <IconButton
+          // MS Co-Pilot gave me the code below, but doing this on my machine results in an
```

Comment:
Add the following
"TODO: Implement email link (https://crestonetech.atlassian.net/browse/CT-45)"

Reply:
done